### PR TITLE
Issue No. #794 [BUG FIXED] In the footer section of the site, the copyright year should be dynamically generated instead of being statically hardcoded.

### LINKED ISSUE #794: [BUG] In the footer section of the site, the copyright year should be dynamically generated instead of being statically hardcoded.
Screenshot 2023-06-15 143401
Screenshot 2023-06-15 143629
Screenshot 2023-06-15 144132

### PR DESCRIPTION
**Pull request by gauravsingh1281**
In this pull request, I made the copyright year dynamic. This means that the copyright year will always be up-to-date without anyone having to change it manually. For example, if the current year is 2023, the copyright section will show 2023. And when the year changes to 2024, the copyright section will automatically update to 2024.

closes #794

### CODE REVIEW

File: shlok_page/motivation.html [hunk 1]
```diff
@@ -1,456 +1,320 @@
...
-          >
+          <a class="nav-link pb-3 hover:text-indigo-600 relative font-semibold after:content-[''] after:bottom-0 after:h-1 after:w-0 after:left-0 after:absolute after:bg-indigo-600 hover:after:w-full after:ease-in-out after:duration-300"
+            href="../Blogs/Blog.html">Blog</a>
         </li>
         <li class="active nav-item">
-          <a
-            class="nav-link nav-link pb-3 hover:text-indigo-600 relative font-semibold after:content-[''] after:bottom-0 after:h-1 after:w-0 after:left-0 after:absolute after:bg-indigo-600 hover:after:w-full after:ease-in-out after:duration-300"
-            href="../Importance of Sloks/Great People.html"
-            >Power of Shloks</a
-          >
```
Comment (Viveksati5143) [inline shlok_page/motivation.html:76]: Missing code from here

File: shlok_page/motivation.html [hunk 2]
```diff
@@ -1,456 +1,320 @@
...
-              class="nav-link pb-3 hover:text-indigo-600 relative font-semibold after:content-[''] after:bottom-0 after:h-1 after:w-0 after:left-0 after:absolute after:bg-indigo-600 hover:after:w-full after:ease-in-out after:duration-300"
-              href="../Blogs/Blog.html"
-              >Blog</a
-            >
-          </li>
-          <li class="active nav-item">
-            <a
-              class="nav-link nav-link pb-3 hover:text-indigo-600 relative font-semibold after:content-[''] after:bottom-0 after:h-1 after:w-0 after:left-0 after:absolute after:bg-indigo-600 hover:after:w-full after:ease-in-out after:duration-300"
-              href="../Importance of Sloks/Great People.html"
-              >Power of Shloks</a
```
Comment (Viveksati5143) [inline shlok_page/motivation.html:172]: ...

**Review by Viveksati5143 — CHANGES_REQUESTED** [review]

File: shlok_page/motivation.html (hunk 2 above)
Comment (gauravsingh1281) [reply shlok_page/motivation.html:172]: maine check kiya aur **MAIN REPO** se bhi refer kiya koi code missing nhi jo pehle missing tha ushe maine solve krdiya ap **functionality aur UI** dekh skte ho dono ka same h

**Comment by gauravsingh1281** [comment]
Screenshot 2023-07-15 140024
Screenshot 2023-07-15 140052

**Review by Viveksati5143 — APPROVED** [review]

**Review by akshitagupta15june — APPROVED** [review]
Good work!

**Pull request merged**